### PR TITLE
Storybook: PoC flow with SFPM, MovableModel and drag&drop

### PR DESCRIPTION
**Pull request by micieslak**
### What does the PR do

Presents flow allowing to maintain ordering using `MovableModel` proxy with combination with SFPM.
This approach is intended to be used in `Profle Showcase` functionality (settings part).

Additionally, the PR fixes small issue in `MovableModel` (improper index was passed to source model in data(...) method)

[Kazam_screencast_00087.webm](https://github.com/status-im/status-desktop/assets/20650004/dbe12f7f-8919-4447-bf95-b3c8254dfa7b)

### Affected areas
`MovableModel`, `Storybook`

### CODE REVIEW

File: ui/StatusQ/src/movablemodel.cpp [hunk 1]
```diff
@@ -88,7 +88,7 @@ QVariant MovableModel::data(const QModelIndex &index, int role) const
     if (m_sourceModel == nullptr)
         return {};
 
-    return m_sourceModel->data(index, role);
+    return m_sourceModel->data(m_sourceModel->index(index.row(), index.column()), role);
```
Comment (caybro) [inline ui/StatusQ/src/movablemodel.cpp:91]:
```suggestion
    return m_sourceModel->index(index.row(), index.column()).data(role);
```

**Review by caybro — APPROVED** [review]
LGTM

Eventhough it's still a POC, I think it's a good time now to introduce at least some basic test(s); afaics we don't have any for the MovableModel, do we?

File: ui/StatusQ/src/movablemodel.cpp (hunk 1 above)
Comment (micieslak) [reply ui/StatusQ/src/movablemodel.cpp:91]: We have tests in `tst_MovableModel.cpp`

**Review by alexjba — APPROVED** [review]
LGTM!

**Pull request merged**